tests/longevity/results/1.0.0.md

```diff
@@ -0,0 +1,234 @@
+```
+
+No entries found.
+
+#### Access Log
+
+Non-200 response codes in NGINX access logs:
+
+```text
+severity=INFO
+"GET" "HTTP/1.1" -"200"
+```
+
+No such responses.
+
+## Key Metrics
+
+### Containers memory
+
+![memory.png](1.0.0-images/memory.png)
+
+No unexpected spikes or drops.
+
+### Containers CPU
+
+![cpu.png](1.0.0-images/cpu.png)
+
+No unexpected spikes or drops.
+
+### NGINX metrics
+
+![stub-status.png](1.0.0-images/stub-status.png)
+
+The drop of _requests_ on Sep 29 is neither significant nor has any correlated errors in NGINX logs.
+
+### Reloads
+
+Rate of reloads - successful and errors:
+
+![reloads.png](1.0.0-images/reloads.png)
+
+Reload spikes correspond to 1 hour periods of backend re-rollouts.
+However, small spikes, like at 1pm Sep 29, correspond to periodic reconciliation of Secrets, which (incorrectly)
+triggers a reload -- https://github.com/nginxinc/nginx-gateway-fabric/issues/1112
+
+A small percentage of reloads finished with error. That happened because of the bug in NFG - it wouldn't wait long
+enough for the NGINX master to start new worker
+processes --  https://github.com/nginxinc/nginx-gateway-fabric/issues/1106
+
+Reload time distribution with 50th, 95th and 99th percentiles and the threshold:
+
+![reload-time.png](1.0.0-images/reload-time.png)
+
+Note - 60s is the threshold for waiting for NGINX to be reloaded.
+
+Reload related metrics at the end:
+
+```text
+# HELP nginx_gateway_fabric_nginx_reloads_milliseconds Duration in milliseconds of NGINX reloads
+# TYPE nginx_gateway_fabric_nginx_reloads_milliseconds histogram
+nginx_gateway_fabric_nginx_reloads_milliseconds_bucket{class="nginx",le="500"} 5608
+nginx_gateway_fabric_nginx_reloads_milliseconds_bucket{class="nginx",le="1000"} 13926
+nginx_gateway_fabric_nginx_reloads_milliseconds_bucket{class="nginx",le="5000"} 14842
+nginx_gateway_fabric_nginx_reloads_milliseconds_bucket{class="nginx",le="10000"} 14842
+nginx_gateway_fabric_nginx_reloads_milliseconds_bucket{class="nginx",le="30000"} 14842
+nginx_gateway_fabric_nginx_reloads_milliseconds_bucket{class="nginx",le="+Inf"} 14842
+nginx_gateway_fabric_nginx_reloads_milliseconds_sum{class="nginx"} 8.645665e+06
+nginx_gateway_fabric_nginx_reloads_milliseconds_count{class="nginx"} 14842
+```
+
+All successful reloads took less than 5 seconds.
+
+## Opened Issues
+
+- NGF doesn't wait long enough for new NGINX workers to
+  start - https://github.com/nginxinc/nginx-gateway-fabric/issues/1106
+- NGF unnecessary reloads NGINX when it reconciles
+  Secrets - https://github.com/nginxinc/nginx-gateway-fabric/issues/1112
+- Statuses not reported because no leader gets elected - https://github.com/nginxinc/nginx-gateway-fabric/issues/1100
+- Use NGF Logger in Client-Go Library - https://github.com/nginxinc/nginx-gateway-fabric/issues/1101
+
+## Future Improvements
+
+- Control Plane transaction weren't fully tested. While we tested that NGF processes EndpointSlices changes, we didn't
```

Comment:
```suggestion
- Control Plane transactions weren't fully tested. While we tested that NGF processes EndpointSlices changes, we didn't
```

nit: some small grammar touches needed for the next couple of lines, can't really figure out what it should be corrected to.